fpga/bigpulp/tcl/ic_wrapper_bd.tcl

```diff
@@ -7,13 +7,20 @@ if { [version -short] == "2016.3" } {
   set XLCONCAT_VERSION "2.1"
   set BLK_MEM_GEN_VERSION "8.3"
   set MB_VERSION "10.0"
-} else { 
+} elseif { 
```

Comment:
The condition is missing here.

Reply:
I could not see the usage of the file in the pulp design I ran. Is it used for some different design?